src/main/java/org/hydev/mcpm/client/database/LocalPluginTracker.java

```diff
@@ -25,7 +26,7 @@
  */
 public class LocalPluginTracker implements PluginTracker {
     // CSV file storing the list of manually installed plugins
-    private String mainLockFile = "plugins/mcpm.lock.csv";
+    private String mainLockFile = "plugins/mcpm.lock.json";
```

Comment:
How did this sneak its way into another PR? LOL
LMK when you revert this.

Reply:
fixed. I just have to rebase from main